### ISSUE TITLE
Booster dose

### ISSUE THREAD

**Issue by Shiaulis** [okestonia/koroonakaart#268]
Hey! Do you have any plans to add statistics for people who got booster shot?

**Comment by neuroactive** [comment]
Yes, there are plans to add statistics on booster shots. The chart for vaccinations will undergo a redesign soon and the new one will include information on booster shots.
